### DISCUSSION TITLE
Merge the Quantity

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#452] [Ideas]
Hi Team,

In the POS screen, when I am trying to add the same quantity two times, it is being added in a new line. In most POS they will increase the quantity based on how many times it is being added, and so it saves space and don't confuse.

![image](https://user-images.githubusercontent.com/16856248/147846761-3a269b37-1a45-4f4e-8f84-e25378b60845.png)

Also having - + button in the quantity area will help to reduce or increase the quantity easily.

![image](https://user-images.githubusercontent.com/16856248/147846777-71b20b4a-ac7e-4598-ac84-fdb49cd12ded.png)

**Comment by Blair2004** [comment]
It's possible to merge similar products on the receipt

![image](https://user-images.githubusercontent.com/5265663/149032789-53b7b116-4d64-4bf0-b852-f8afb408c6d3.png)

We don't like that - and + option. 

Thank you for your ideas.
